apps/sms/test/unit/information_test.js

```diff
@@ -1067,10 +1060,6 @@ suite('Information view', function() {
     });
     test('view status before show method', function() {
       sinon.assert.calledWith(groupView.renderContactList, participants);
```

Comment:
In this case maybe we need to chage the assertion from ThreadUI.setHeaderContent to setAttribute